src/main/kotlin/net/maizegenetics/phgv2/cli/SetupEnvironment.kt

```diff
@@ -39,22 +60,21 @@ class SetupEnvironment : CliktCommand(help="create conda environment for PHG") {
                 .filter { it.contains("prefix already exists") }
                 .toList()
             if (success == null || success.size == 0) {
-                println("conda env create command for file ${envFile} run via ProcessBuilder returned error code $error")
+                myLogger.severe("conda env create command for file ${envFile} run via ProcessBuilder returned error code $error")
```

Comment:
What does severe mean? Is that different than error?

Reply:
For the logger used, there is no "error".  "Severe" is the only option above warning.  If I'm using the wrong logger, let me know and I'll change that.

Reply:
You're using the wrong logger

import org.apache.logging.log4j.LogManager
import org.apache.logging.log4j.Logger